opus/src/components/singnup/SignUp.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import logo from './logo-OPUS.png';
+import './signUp.css';
+
+export default class SignUp extends Component {
+  render() {
+    return (
+      <div>
+        <img className="logoCommunOpus" src={logo} alt="logo OPUS"></img>
+
+        <form className="containerFormSignUp">
+          <input className="inputSignUP" type="text" placeholder="Nom" />
+          <input className="inputSignUP" type="text" placeholder="Prénom" />
+          <input className="inputSignUP" type="text" placeholder="Email" />
+          <input
+            className="inputSignUP"
+            type="text"
+            placeholder="Mot de passe"
+          />
```

Comment:
you may consider other input types: https://www.w3schools.com/tags/att_input_type.asp